src/services/cache.service.ts

```diff
@@ -0,0 +1,61 @@
+import config from 'config';
+import logger from 'logger';
+import redis, {RedisClient} from 'redis';
+
+import {OktaUser} from 'services/okta.interfaces';
+
+export default class CacheService {
+    private static getCacheClient(): RedisClient {
+        const client: RedisClient = redis.createClient({ url: config.get('redis.url') as string });
```

Comment:
this creates a new connection to redis on each request. Is there any way we can keep that connection alive for the lifespan of the node app, so we reduce load on redis and RTT?

Reply:
should be fixed